src/tlog-test-json-stream-btoa.c

```diff
@@ -49,7 +49,7 @@ main(void)
                     "%02hhx %02hhx %02hhx %02hhx\n",
                     i,
                     btoa_buf[0], btoa_buf[1],
-                    btoa_buf[2], btoa_buf[3], 
+                    btoa_buf[2], btoa_buf[3],
```

Comment:
Please remove this unnecessary whitespace change.

Reply:
This still applies. Please do not remove this extra whitespace in this commit, it's unrelated to the changes you're doing. Make a separate commit cleaning it up (join it with the other unrelated whitespace change above).